server/src/main/java/io/unitycatalog/server/persist/dao/CatalogInfoDAO.java

```diff
@@ -28,6 +30,10 @@ public class CatalogInfoDAO {
     @Column(name = "comment")
     private String comment;
 
+    @Column(name = "properties")
+    @JdbcTypeCode(SqlTypes.JSON)
+    private Map<String, String> properties;
```

Comment:
The way properties work is that they are stored in a different table constructed by PropertyInfoDAO, we should use the same structure instead of adding a new column. These are references of how it is done [created](https://github.com/unitycatalog/unitycatalog/blob/main/server/src/main/java/io/unitycatalog/server/persist/CatalogRepository.java#L53) and [deleted](https://github.com/unitycatalog/unitycatalog/blob/main/server/src/main/java/io/unitycatalog/server/persist/CatalogRepository.java#L194).